cmd/promtool/main.go

```diff
@@ -20,8 +20,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
If we're going to migrate we should complete the discussion elsewhere and then migrate. Introducing thing only in new code has a tendency to end up with a split world for rather a long time.